README.md

```diff
@@ -100,34 +100,57 @@ To get started with DCLM, follow these steps:
 3. **Set up your environment**:
     DCLM uses AWS for storage and possible as a compute backend, and ray for distributed processing.
     Ensure you have the necessary environment variables and configurations for AWS and Ray clusters.
-    
+
     We recommend the use of Python 3.10 with DCLM.
 
-## Exp Data
-This repository contains the exp_data folder to help keep track of and help run experiments. exp_data gives a record of the datasets created, models trained, and evaluations finished. Python based code automatically generates json files after tokenize shuffle, training, and evaluation. If needed, you can manually create a json by copying the format of one of the existing files and modifying the relevant fields. It is key to change the uuid (use python uuid to generate) because that is the unique identifier the code uses to identify each json. Paths in the json may be local as long as they are accessible when the json is used.
+## Workflow Overview and Exp Data
+
+At a high-level, the DCLM workflow consists of five main steps: (1) raw source selection; (2) data processing; (3) tokenization and shuffling; (4) model training; (5) model evaluation. To be eligible for our leaderboards, you are required to use our scripts for steps (3)-(5).
+
+To help keep track of and run experiments, this repository uses the [exp_data](exp_data) folder to log a record of the inputs and outputs of these steps (e.g., datasets created, models trained, and evaluations finished). Each record is called a "reference JSON" and can be thought of as an ID card for a particular input or output asset. Overall, there are four categories of reference JSONs:
+
+- **Untokenized Datasets:** these refer to datasets in their raw text form which can either be the inputs or outputs of data processing ([exp_data/datasets/raw_sources](exp_data/datasets/raw_sources) and [exp_data/datasets/untokenized](exp_data/datasets/untokenized)).
```

Comment:
add respectively

Reply:
reasoning for not adding this is that a user can also use a dataset in the untokenized folder as a starting point (e.g., if they want to run two different processing steps)

Reply:
Ok makes sense!